Tasks/PublishSymbols/IndexHelpers/IndexFunctions.ps1

```diff
@@ -1,3 +1,14 @@
+function Get-PdbstrPath {
+    [CmdletBinding()]
+    param()
+
+    Trace-VstsEnteringInvocation $MyInvocation -Parameter @()
+    $pdbstrPath = [System.IO.Path]::GetFullPath("$PSScriptRoot\..\pdbstr.exe")
+    Assert-VstsPath -LiteralPath $pdbstrPath -PathType Leaf -PassThru
```

Comment:
Same comment here about `-passthru`, it looks like the caller will get an array with two items. Also the comment applies further below too.